src/store/channels/saga.ts

```diff
@@ -32,7 +32,8 @@ export function* markAllMessagesAsRead(channelId, userId) {
     return;
   }
 
-  const status = yield call(markAllMessagesAsReadAPI, channelId, userId);
+  const chatClient = yield call(chat.get);
+  const status = yield call([chatClient, chatClient.markRoomAsRead], channelId, userId);
 
   if (status === 200) {
```

Comment:
Seems like we need to generalize this status check now? Maybe the `markRoomAsRead` should return a boolean rather than http status code?